generator/generator.go

```diff
@@ -243,16 +243,16 @@ func generateInitRamfs(conf *generatorConfig) error {
 // appendCompatibilitySymlinks appends symlinks for compatibility with older firmware that loads extra files from non-standard locations
 func appendCompatibilitySymlinks(img *Image) error {
 	symlinks := []struct{ src, target string }{
```

Comment:
So you are basically using relative symlinks. It is better to change the commit description "Tweaked symlinks" -> "Use relative symlinks".